### DISCUSSION TITLE
floor

### DISCUSSION THREAD

**Discussion by something-iscringe** [betting-iscringe/betting-iscringe.github.io#1] [General]
I'm not making a discord

**Comment by vt-idiot** [comment]
ohayo, uh, I forgot how this worked

**Reply by something-iscringe** [reply]
the readme in `data/script` would help you run [the script](https://github.com/betting-iscringe/data/blob/f5dd29a1dd93c37389c4752ffa4bf64d3a3ab6bd/script/getter.py), the important things would be the index.js files in each folder (including the root) and the generated file

**Comment by something-iscringe** [comment]
@vt-idiot hey, so I just created a [github action(currently disabled)](https://github.com/betting-iscringe/data/actions/workflows/update-indices.yml) that would auto update `<category>/index.json` after anybody adds a new file into the folders, check [these](https://github.com/betting-iscringe/data/commit/a4ab3be0be9efec87a280c5370456ce44cae014a) [commits](https://github.com/betting-iscringe/data/commit/c880fb4526c4b1eb087f200b7faf5fbdf6d7ef15).
It pushes to master by default which would make updating via cli annoying but I can make it push to another branch and remove `<category>/index.json` from master if you like

**Reply by something-iscringe** [reply]
pinging @JeanneDGames too, thoughts?

**Reply by JeanneDGames** [reply]
Doesn't make a difference to me. Right now I just manually index.json whenever I add a new file.

**Reply by something-iscringe** [reply]
@JeanneDGames alright, I'll enable the action for now if anybody doesn't like it they can disable it then kek

**Comment by something-iscringe** [comment]
@JeanneDGames I added an action so that you can swap defaults via github ui instead of manually editing the file. A bit hard to find at first but here's the direct link
https://github.com/betting-iscringe/data/actions/workflows/update-defaults.yml
![image](https://github.com/user-attachments/assets/bb9fcf55-24d2-43fc-8502-fcce86ff3ebe)
